src/librustdoc/clean/mod.rs

```diff
@@ -2793,7 +2793,7 @@ fn clean_maybe_renamed_item<'tcx>(
             ItemKind::Fn(ref sig, generics, body_id) => {
                 clean_fn_or_proc_macro(item, sig, generics, body_id, &mut name, cx)
             }
-            ItemKind::Trait(_, _, generics, bounds, item_ids) => {
+            ItemKind::Trait(_, generics, bounds, item_ids) => {
```

Comment:
can you create an issue about changing the rustdoc auto trait output to something like what Ralf suggested on zulip, just having a short text that the trait is implemented automatically?

Reply:
#116268